test/support/poro.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveModelSerializersWithoutLegacyModelSupport
+  module_function
+
+  def poro_without_legacy_model_support(superklass = ActiveModelSerializers::Model, &block)
```

Comment:
An example test helper [I derived from one I wrote in some unmerged commits](https://github.com/rails-api/active_model_serializers/blob/ed1904358767697883f6832ac4c9eb833b82978a/test/support/poro.rb)